wiki/How_You_Can_Help!/en.md

```diff
@@ -1,77 +1,77 @@
+- Browse the [Tournaments](https://osu.ppy.sh/community/forums/55) forum; tournaments take time and a lot of manpower to organize without hitches, and you can help out by being a spectator judge in some cases.
+- Check out what's going on in the [Development](https://osu.ppy.sh/community/forums/2) forum; it is where most serious projects happen.
 - Join a [beatmap project](https://osu.ppy.sh/community/forums/53) and learn how fun collaborating with others can be!
-- Start your own project! Whether it is bringing back dancing pippi, or just making something cool for osu!, anything is accepted.
+- Start your own project! Whether it's bringing back dancing Pippi, or just making something cool for osu!; anything is accepted!
 
 ## Contribute to the future of osu!
 
-Started in 2017, the osu!lazer project is the current focus of the osu! team, along with the new website, which you see today.
+Started in 2017, the osu!lazer project is the current focus of the osu! team, along with the new website, which is what you see today! <!-- I personally don't like this line: it's oddly short and doesn't really finish what it started. But I don't really know what else to put here. Maybe a link to a page explaining what osu!lazer is at the very least. If any of you guys have ideas, please tell me. -toaster -->
 
-You can help make them even better, and make us closer to the future of osu! by helping on these areas:
+If you would like to help make these projects even better and bring us closer to the future of osu!, then please consider…
 
-- Helping with the [osu!lazer](https://github.com/ppy/osu) project. There is much to implement on the new version of osu!, and you can help make it happen!
+- Helping with the [osu!lazer](https://github.com/ppy/osu) project. There is so much to implement on the new version of osu!, and you can help make it happen!
 - Make the [new website](https://github.com/ppy/osu-web) even better.
 
 ## Avoid being negative! Make people happy!
 
-There is an old saying that you should become part of the solution, not the problem. You can help on osu! by making the community a brighter place. People complain, but there are many ways you can help lighten up the mood in the community:
+There is an old saying that you should become part of the solution, not the problem. Simply by following this old saying, you'd be helping osu! by making the community a brighter place. People will always complain, but there are still so many ways you can help lighten up the mood in the community:
 
-- Be friendly to each other. Life is too short to argue.
-- [Report cheaters when you see them](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play). Do not complain about them on other mediums, a simple `!report <username> seems to be cheating` can fix your problems within minutes to hours.
-- Go out of your way to mediate negative situations and turn them into positive outcomes.
+- Be friendly to each other; life is too short to argue.
```

Comment:
This is based on peppy's original statement, I don't know if this has the same effect as the original.

Reply:
The only change I made was switching the period to a semicolon because the two clauses are closer than a period would suggest.

But I can't tell if that change deleted the other stuff (I'm on mobile atm). If that's what you're referring to, then yeah, I'll keep the original text.